calicoctl/commands/node/status.go

```diff
@@ -0,0 +1,159 @@
+// Copyright (c) 2016 Tigera, Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package node
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"net"
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/docker/engine-api/client"
+	"github.com/docker/engine-api/types"
+	"github.com/docopt/docopt-go"
+	"golang.org/x/net/context"
+
+	"github.com/olekukonko/tablewriter"
+)
+
+// Check for Word_<IP> where every octate is seperated by "_", regardless of IP protocols
+// Example match: "Mesh_192_168_56_101"
+var bgpPeerRegex, _ = regexp.Compile(`\w+_\d+_\d+_\d+_\d+`)
+
+// Status prings status of the node and returns error (if any)
+func Status(args []string) error {
+	doc := `Usage:
+calicoctl node status
+
+Description:
+  Display the status of the Calico node`
+
+	// Note: This call is ignoring the error because error check happens at the level above
+	// i.e at `node.go` before it calls `node.Status`. This call is just so help message gets
+	// printed for this option
+	_, _ = docopt.Parse(doc, args, true, "calicoctl", false, false)
+
+	ctx := context.Background()
+
+	if os.Getenv("DOCKER_HOST") == "" {
```

Comment:
Not sure I understand this bit.  Why are we checking DOCKER_HOST to determine if we should set DOCKER_API_VERSION?